pkg/apis/migration/v1alpha1/verify.go

```diff
@@ -0,0 +1,248 @@
+package v1alpha1
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	runtime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Types
+const (
+	UnhealthySourceNamespaces      = "UnhealthySourceNamespaces"
```

Comment:
The migration controller verification only deals with the destination.  No need for Source/Destination differentiation or for both.  Just `UnhealthyNamespaces`.